packages/protocol/test/governance/network/epochrewards.ts

```diff
@@ -64,9 +64,9 @@ contract('EpochRewards', (accounts: string[]) => {
   const nonOwner = accounts[1]
 
   const targetVotingYieldParams = {
-    initial: toFixed(new BigNumber(1 / 20)),
-    max: toFixed(new BigNumber(1 / 5)),
-    adjustmentFactor: toFixed(new BigNumber(1 / 365)),
+    initial: toFixed(new BigNumber(0.00016)),
+    max: toFixed(new BigNumber(0.0005)),
+    adjustmentFactor: toFixed(new BigNumber(0.00000112799)),
```

Comment:
I am wondering how we can leave a hint on how to choose these parameters, especially the adjustmentfactor, in an economically reasonable way. How do you feel about scheduling some time in one of the next milestones to add a "parameter overview" section to docs.celo.org that explains the economic meaning of those parameters and links to background material like the doc you wrote up on the adjustmentFactor and its fallacies?

Reply:
Good idea, that would be useful. Here is a ticket, feel free to edit: https://app.zenhub.com/workspaces/platform-economics---backlog-board-600a9f7399b43a001348ae1b/issues/celo-org/celo-monorepo/8578

Reply:
🙏